### PR TITLE
[native] Add support for background CPU time stats in Shuffle operators

### PR DESCRIPTION
**Pull request by patrickstuedi**
Folds per operator background CPU into OperatorStats.finishTiming
before sending the metrics to the Presto Coordinator. Moreover, enables
collecting of background CPU time in ShuffleWrite operator.

### CODE REVIEW

File: presto-native-execution/presto_cpp/main/operators/tests/UnsafeRowShuffleTest.cpp [hunk 1]
```diff
@@ -40,6 +40,8 @@ namespace facebook::presto::operators::test {
 
 namespace {
 
+static inline const uint64_t kFakeBackgroundCpuTimeMs = 123;
```
Comment (xiaoxmeng) [inline presto-native-execution/presto_cpp/main/operators/tests/UnsafeRowShuffleTest.cpp:43]:
```
s/static inline const/static const/
```

**Review by xiaoxmeng — APPROVED** [review]
@patrickstuedi LGTM. Thanks!

File: presto-native-execution/presto_cpp/main/operators/tests/UnsafeRowShuffleTest.cpp (hunk 1 above)
Comment (patrickstuedi) [reply presto-native-execution/presto_cpp/main/operators/tests/UnsafeRowShuffleTest.cpp]: Done.

**Review by mbasmanova — COMMENTED** [review]
@patrickstuedi Patrick, thank you for the change. Would you update commit message to make sure each line is wrapped at 80 characters?

**Review by mbasmanova — APPROVED** [review]
Looks great. Maybe split into 2 commits.

[native pos] Collect background CPU time in shuffle operators
[native] Fold background CPU time into finish CPU time

**Pull request merged**